test/conformance_test.jl

```diff
@@ -177,12 +179,12 @@ function test_GroupElement_interface(g::GEl, h::GEl) where {GEl<:GroupElement}
             @test similar(g) isa typeof(g)
         end
 
-        one!, inv!, mul!, conj!, comm!, div_left!, div_right! = (
+        one!, inv!, mul!, conj!, commutator!, div_left!, div_right! = (
```

Comment:
So... to color the bike shed... it is inv, mul, div... why not expand those? Because Julia defines them, I guess... but then `comm` fits better into that sequence. OTOH if we only expand things that we add over Base, shouldn't we so it for all and also expand conj?

Don't get me wrong, I am not opposed, but I *am* confused: what's the rule here?

Reply:
tbh. I have no idea, but this change was requested by @thofma in #20 ...